### PR TITLE
Re-introduce a Ox::Raw type

### PR DESCRIPTION
**Pull request by cade**
Ref: https://github.com/ohler55/ox/pull/100

I ran into the same use case as @tim-vandecasteele today and found his abandoned PR, so I thought I'd pick it up and try to drive it to completion.

I'd like to be able to:

```ruby
doc = Ox::Document.new(version: '1.0', encoding: 'UTF-8')
ox_root = Ox::Element.new(:projects)
doc << ox_root

@user.projects.each do |project|
  key = "project-#{project.id}-#{project.updated_at.to_i}"
  project_xml = Rails.cache.read key
  if project_xml
    ox_root << Ox::Raw.new(project_xml)
  else
    ox_project = Ox::Element.new(:project)
    ox_project << (Ox::Element.new(:id) << project.id.to_s)
    # ... populate ox_project with other attributes
    ox_root << ox_project
    Rails.cache.write key, Ox.dump(ox_project)
  end
end

Ox.dump doc
```

Let me know if you need anything else from me or have any questions/suggestions! Ox is *awesome*, and thanks very much for your work! :heart: 

(Tim: I hope you don't mind that I amended your commit to clean it up. I wanted you to have credit without adding "revert whitespace" commits.)

### CODE REVIEW

File: lib/ox/raw.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+module Ox
+  # Raw elements are used to inject existing XML strings into a document
+  class Raw < Node
```
Comment (ohler55) [inline lib/ox/raw.rb]: Please include a caveat that use of this feature can result in invalid XML.

**Comment by ohler55** [comment]
Other than a warning that you can shoot yourself in the foot I'll merge when the comment is updated. Thanks.

**Comment by cade** [comment]
Comment added. Thanks for looking so quickly!

**Comment by ohler55** [comment]
I'll get this in a release this week.

**Pull request merged**